mapfishapp/src/main/webapp/app/js/GEOR_mapinit.js

```diff
@@ -340,6 +350,67 @@ GEOR.mapinit = (function() {
                 }
             }
         });
+        
+        // support for JSON search parameter
+        if( initSearch.hasOwnProperty('typename') && initSearch.hasOwnProperty('featurens') 
+            && initSearch.hasOwnProperty('owsurl') && initSearch.hasOwnProperty('cql_filter') ) {
+            var record = {
+                typeName: initSearch.typename,
+                featureNS: initSearch.featurens,
+                owsURL: initSearch.owsurl,
+                WFSversion : "1.1.0"
+            }
+            var filter = (new OpenLayers.Format.CQL()).read(initSearch.cql_filter);
+            
+            var attStore = GEOR.ows.WFSDescribeFeatureType(record, {
```

Comment:
I borrowed only a little part (3 lines) from the querier for get the geometryName do you think it really need to define a function ?

The callbacks I use are not the same as the querier.